src/utility/Filters.java

```diff
@@ -19,4 +26,40 @@ public static SubCategory filterSubCategoryByCode(List<SubCategory> subCategory,
                 .findFirst().orElse(null);
     }
 
+    public static List<Category> activeCategory(List<Category> categories) {
```

Comment:
O que acha de um nome mais descritivo e no plural:
```suggestion
    public static List<Category> findActiveCategories(List<Category> categories) {
```